.github/workflows/pkg-platform.yml

```diff
@@ -135,6 +135,13 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: mock `aws`
+        if: inputs.dry-run
+        run: |
+          echo '#!/bin/sh' > /usr/local/bin/aws
+          echo 'echo "$@"' >> /usr/local/bin/aws
+          chmod +x /usr/local/bin/aws
+
```

Comment:
won't have to worry about path issues (or polluting our builders!!!) if you just set `AWS` in github_env, and replace lower calls with `$AWS`.

Reply:
notably this only ran on `ubuntu-latest` so was probs safe